### PR TITLE
Rename "tags" to "attributes"

### LINKED ISSUE #598: Rename "tags" to "attributes"
Span "tags" is leftover terminology from OpenTracing, we should update to use OpenTelemetry terminology span "attributes".

### PR DESCRIPTION
**Pull request by malafeev**
fix #598

### CODE REVIEW

File: agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java [hunk 1]
```diff
@@ -83,13 +84,13 @@ public Span onPeerConnection(final Span span, final InetSocketAddress remoteConn
       } else {
         // Failed DNS lookup, the host string is the name.
         final String hostString = remoteConnection.getHostString();
-        span.setAttribute(MoreTags.NET_PEER_NAME, hostString);
+        span.setAttribute(SemanticAttributes.NET_PEER_NAME.key(), hostString);
```
Comment (trask) [inline agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java:87]: we should use `SemanticAttributes` like this everwhere, we can do that in separate PR though, probably good to merge this PR quickly to avoid conflicts

```suggestion
        SemanticAttributes.NET_PEER_NAME.set(span, hostString);
```

Comment (malafeev) [reply agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java:87]: @trask if PR will be merged I will work on another one to use `SemanticAttributes.NET_PEER_NAME.set(span, hostString);`

Comment (anuraaga) [reply agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java:87]: Yeah if I recall correctly we're waiting for the SDK version bump to have the form that supports setting into a builder

Comment (malafeev) [reply agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java:87]: anybody works on SDK version bump? looking for it.

Comment (anuraaga) [reply agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/BaseDecorator.java:87]: I'm guessing @iNikem is based on https://gitter.im/open-telemetry/opentelemetry-java-instrumentation?at=5f034b32fa0c9221fc818e08
